README.md

```diff
@@ -1,39 +1,11 @@
 Symbolize (attribute values)
 =========
 
-# No longer maintained
-
-
-## Please check this other gems:
-
-# Mongoid
-
-https://github.com/thetron/mongoid-enum
-
-
-# ActiveModel & POROs
-
-https://github.com/brainspec/enumerize
-
-
-This plugin introduces an easy way to use symbols for values of attributes.
-Symbolized attributes return a ruby symbol (or nil) as their value
-and can be set using :symbols or "strings".
-
-
-
-Install
--------
-
-Gem
-
-
-    gem install symbolize
-
-
-Gemfile
+Notes
+-----
 
-    gem 'symbolize'
+This is a fork. For more details see the original repo:
+http://github.com/nofxx/symbolize
```

Comment:
Sort of? This repo was originally a fork of that. Clearly the two have diverged quite a bit. I’m in favor of leaving it in, since at least it gives a clear idea of where this started.

I’m also going to work to chop this out of Cowboy as we upgrade a few other things. Mongoid has native (and better) support for most of the functionality this provides.